docs/source/share_kedro_viz.md

```diff
@@ -39,6 +39,29 @@ pip install -r src/requirements.txt
 
 You can host your Kedro-Viz project on Amazon S3. You must first create an S3 bucket and then enable static website hosting. To do so, follow the [AWS tutorial](https://docs.aws.amazon.com/AmazonS3/latest/userguide/HostingWebsiteOnS3Setup.html) to configure a static website on Amazon S3.
 
+
+Once the s3 bucket is created, you'll need to creat an (Identity and Access Management) IAM user account, user group, and access keys:
+
+Sign in to the [AWS Management Console](https://console.aws.amazon.com/s3/) and create an  IAM user account.
+For more information, see the official AWS documentation about [IAM Identities](https://docs.aws.amazon.com/IAM/latest/UserGuide/id.html).
+
+![](../meta/images/kedro_viz_share_credentials1.png)
+
+Create a user group from the IAM dashboard (ensuring the user group has full access to AWS S3 policy).
+
+![](../meta/images/kedro_viz_share_credentials2.png)
+
+"A user group is a collection of IAM users. User groups simplify permissions management because you can grant, change, and remove permissions for multiple users at the same time". For more information, see the official AWS documentation about [IAM user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html?icmpid=docs_iam_help_panel).
```

Comment:
I don't think we need the copy/paste parts of the AWS documentation into ours.

```suggestion
For more information, see the official AWS documentation about [IAM user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html?icmpid=docs_iam_help_panel).
```

Reply:
Agreed.